src/business-logic/facades/service-facade.ts

```diff
@@ -39,15 +40,13 @@ export class ServiceFacade {
       TimeoutCallbackScheduler.getInstance(LoggerFacade.createLogger()),
       BlueprintsFacade.createBlueprint()
     )
+
+    return RundownLockService.getInstance(rundownTimelineService)
   }
 
   public static createTimelineBuilder(): TimelineBuilder {
     const superflyTimelineBuilder: TimelineBuilder = new SuperflyTimelineBuilder(new JsonObjectCloner())
-    return new BlueprintTimelineBuilder(
-      superflyTimelineBuilder,
-      RepositoryFacade.createConfigurationRepository(),
-      BlueprintsFacade.createBlueprint()
-    )
+    return new BlueprintTimelineBuilder(superflyTimelineBuilder, RepositoryFacade.createConfigurationRepository(), BlueprintsFacade.createBlueprint())
```

Comment:
Was this a linter that changed this?
Because this formatting makes the line way too long.

Comment applies for the other places in this file as well.

Reply:
Return in to previous formation